ktane/modules/buttons.py

```diff
@@ -0,0 +1,139 @@
+﻿from enum import Enum
+from ktane import Module
+
+
+class Color(Enum):
+    white = 1
+    blue = 2
+    red = 3
+    yellow = 4
+
+
+class Text(Enum):
+    detonate = 1
+    hold = 2
+    abort = 3
+
+
+class Indicator(Enum):
+    SND = 1
+    CLR = 2
+    CAR = 3
+    IND = 4
+    FRQ = 5
+    SIG = 6
+    NSA = 7
+    MSA = 8
+    TRN = 9
+    BOB = 10
+    FRK = 11
+
+
+# Module for "The Button". The logic for lit indicators
+# and batteries is likely better placed outside of this
+# class and into a more general space.
+# Quitting in the middle of an event is also not functional because you can't
+# correctly unravel the stack with that input unless you account for it
+# with every function call (which defeats the purpose).
+class Buttons(Module):
+
+    # Number of Batteries
+    batteries = -1
```

Comment:
I don't think that you need to make these declarations here since setting self.whatever inside a method should be enough.